egs/babel_multilang/s5/run-4-anydecode-langs.sh

```diff
@@ -29,10 +28,14 @@ ivector_suffix=
 iter=final
 
 # params for extracting bn features
-multidir=exp/nnet3/multi_bnf_sp
+use_bnf=false # If true, bottleneck feature is extracted and appended as input
+              # for nnet3 model.
```

Comment:
you should probably clarify that if you set it to true, you have to set 'bnf_node_name'.
if you have a script that trains a bottleneck model, why not set bnf_node_name to the bottleneck node?

Reply:
Do you mean to hardcode it in the script? 
People can use this script to extract feature for bottlenek layer after affine, nonlinearity or renorm layer. So bnf_node_name can be tdnn_bn.affine, tdnn_bn.renorm or tdnn_bn.relu.
Also people can use this script to extract non-bottleneck layer and probably do dimensionality reduction on that using PCA.